config/crd/bases/cni.tanzu.vmware.com_antreaconfigs.yaml

```diff
@@ -153,6 +153,18 @@ spec:
                             description: Flag to enable/disable service external IP
                             type: boolean
                         type: object
+                      flowExporter:
+                        description: Specifies FlowExporter related configuration.
+                        properties:
+                          activeFlowTimeout:
+                            type: string
+                          collectorAddress:
+                            type: string
+                          idleFlowTimeout:
+                            type: string
+                          pollInterval:
+                            type: string
+                        type: object
```

Comment:
These seem to be valid but unrelated changes.  Are you purposefully including them or did they slip in silently in a `git add .` command?

(I personally am of the school of thought that different changes should be part of different PRs/commits, but that is just my opinion)

Reply:
Thanks. I will add it as part of separate commit. I think it was generated when I was testing `make generate` and `make manifest`. And I agree this should be added as part of separate commit. I will update the PR.